go/client/cmd_configure_nix.go

```diff
@@ -0,0 +1,378 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin,!windows
+
+package client
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/install"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+)
+
+func NewCmdConfigure(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "configure",
```

Comment:
Yeah tough question.  I guess `system-configure` is ok, but a little unwieldy.  I'd be ok with just `keybase autostart enable/disable` and `keybase fs redirector enable/disable` too.

Reply:
How about sys-configure? I also wanted to keep all these options in one place.

Reply:
Sure that's fine.  I consider @patrickxb more of the CLI guru than me, so if he doesn't register here with an objection, it's fine by me.

Reply:
What about underneath `keybase ctl`?

Reply:
I thought `ctl` was more of a process manager kind of command rather than  system configuration. I don't really care too much though, I can add it there if you think it's better. Would it be `keybase ctl autostart enable` or `keybase ctl configure autostart enable`?

Reply:
do you want users to use this or is it just called by something else?

Reply:
I want users to use it but it'll only be a small number of them that want more control over Keybase. Should not be called by scripts.

Reply:
`keybase ctl autostart enable` sounds good to me.  i don't think we need another top-level command if it's just for a few users.

Reply:
All right, thanks for the advice